hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide3.java

```diff
@@ -1050,13 +991,6 @@ public RegionScanner postScannerOpen(final ObserverContext<RegionCoprocessorEnvi
 
   @Test
   public void testScanWithBatchSizeReturnIncompleteCells() throws IOException {
-    TableName tableName = TableName.valueOf(name.getMethodName());
-    TableDescriptor hd = TableDescriptorBuilder.newBuilder(tableName)
-        .setColumnFamily(ColumnFamilyDescriptorBuilder.newBuilder(FAMILY).setMaxVersions(3).build())
```

Comment:
Did this test not actually rely on the behavior of setting 3 max versions?

Reply:
Interestingly it passes with storing only a single cell version. I revert it anyway to use 3 versions.